tests/conftest.py

```diff
@@ -88,6 +84,17 @@ def pytest_addoption(parser):
         action="store",
         default="../xtgeo-testdata",
     )
+    parser.addoption(
+        "--generate-plots",
```

Comment:
Should we perhaps reverse this? And have a flag to disable it instead? I guess the long term goal is to remove all the plotting, so at some point it will become irrelevant, but while it exists it should perhaps default to on?

Reply:
I thought it should be false by default as there are no asserts for these plots, rather there are images generated. If you want to check that the corresponding function is correct you have to run the test with `--generate-plots` and go look at the generated file, so some manual intervention is necessary.

Reply:
Ok, that makes sense, but could you also write it in a comment, or add it to the commit body?

Reply:
or, perhaps better, just expand the `help`

Reply:
Added both to the option help text and to the commit body.